src/lib/pickles/step_verifier.ml

```diff
@@ -229,7 +229,7 @@ struct
     with_label __LOC__ (fun () ->
         Ops.scale_fast2 p s ~num_bits:Field.size_in_bits )
 
-  let check_bulletproof ~pcs_batch ~(sponge : Sponge.t) ~xi
```

Comment:
It is weird that it is unused, but ok. I changed on a local branch to `let check_bulletproof ~pcs_batch:_foo` and the compiler does not complain. Interesting.